recipe/build-sysroot.sh

```diff
@@ -2,22 +2,5 @@
 
 mkdir -p ${PREFIX}/${target_machine}-conda-linux-gnu/sysroot
 pushd ${PREFIX}/${target_machine}-conda-linux-gnu/sysroot > /dev/null 2>&1
-cp -Rf "${SRC_DIR}"/binary/* .
-mkdir -p usr/include
-cp -Rf "${SRC_DIR}"/binary-tzdata/* usr/
-cp -Rf "${SRC_DIR}"/binary-kernel-headers/include/* usr/include/
-cp -Rf "${SRC_DIR}"/binary-glibc-headers/include/* usr/include/
-cp -Rf "${SRC_DIR}"/binary-glibc-devel/* usr/
-cp -Rf "${SRC_DIR}"/binary-glibc-common/* .
-
-mkdir -p lib
-mv lib64/* lib
-rm -rf lib64
-
-mv usr/lib64/* usr/lib/
-rm -rf usr/lib64
-
-ln -s $PWD/lib $PWD/lib64
-ln -s $PWD/usr/lib $PWD/usr/lib64
-
+cp -Rf ${SRC_DIR}/x86_64-conda_cos6-linux-gnu/sysroot/* .
```

Comment:
This has way more stuff than just the sysroot.

Reply:
Happy to mark this broken and try again. I was trying to test it live.

Reply:
I added the broken label and will make another PR. What do we need to include and/or exclude here?

Reply:
Can you compare with the CDT builds? There are stuff like `libgcc-ng`, `libgomp.spec`, etc.

Reply:
Yes. I will compare and exclude what is not needed.

Reply:
Can we go with https://github.com/conda-forge/ctng-compilers-feedstock/pull/21 if we are splitting the sysroot like this?

Reply:
Yup. Let's decide amongst the options and then clean up the PR we want.